HacktoberfestProject.Web/Data/Configuration/TableConfiguration.cs

```diff
@@ -1,11 +1,6 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace HacktoberfestProject.Web.Data.Configuration
-{
-	public class TableConfiguration
+﻿namespace HacktoberfestProject.Web.Data.Configuration
```

Comment:
What's the change here? Just formatting?

Reply:
Removed the unused using statements